interface/weno/adminClass.php

```diff
@@ -0,0 +1,723 @@
+                        `Accept_RefillResponse` varchar(5) DEFAULT NULL,
+                        `Accept_RxChangeResponse` varchar(5) DEFAULT NULL,
+                        `Accept_Verify` varchar(5) DEFAULT NULL,
+                        `Accept_CancelRx` varchar(5) DEFAULT NULL,
+                        `Accept_RxHistoryRequest` varchar(5) DEFAULT NULL,
+                        `Accept_RxHistoryResponse` varchar(5) DEFAULT NULL,
+                        `Accept_Census` varchar(5) DEFAULT NULL,
+                        `Accept_Resupply` varchar(5) DEFAULT NULL,
+                        PRIMARY KEY (`id`)
+                        ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=11150"
+    		);
+
+          sqlStatement(
+            "CREATE TABLE IF NOT EXISTS `prescription_rx_log` (
+                         `id` int(20) NOT NULL AUTO_INCREMENT,
+                         `prescription_id` int(6) NOT NULL,
+                         `date` varchar(25) NOT NULL,
+                         `time` varchar(15) NOT NULL,
+                         `code` int(6) NOT NULL,
+                         `status` text NOT NULL,
+                         `message_id` varchar(100) DEFAULT NULL,
+                         `read` int(1) DEFAULT NULL,
+                         PRIMARY KEY (`id`)
+                          ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=133" 
+          	);
+
+          sqlStatement(
+              "CREATE TABLE IF NOT EXISTS `narcotics` (
+                            `id` int(11) NOT NULL AUTO_INCREMENT,
+                            `drug` varchar(255) NOT NULL,
+                            `dea_number` varchar(5) NOT NULL,
+                            `csa_sch` varchar(2) NOT NULL,
+                            `narc` varchar(2) NOT NULL,
+                            `other_names` varchar(255) NOT NULL,
+                             PRIMARY KEY (`id`)
+                           ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=570"
+          	);
+
+          sqlStatement("ALTER TABLE `users` ADD `weno_prov_id` VARCHAR(15) NULL AFTER `newcrop_user_role`");
+          
+          sqlStatement("ALTER TABLE `prescriptions` ADD `ntx` INT(2) NULL DEFAULT NULL AFTER `prn`, ADD `rtx` INT(2) NULL DEFAULT NULL AFTER `ntx`, ADD `txDate` DATE NOT NULL AFTER `rtx`");
+        
+          sqlStatement("ALTER TABLE `pharmacies` ADD `ncpdp` INT(12) NULL AFTER `email`, ADD `npi` INT(12) NULL AFTER `ncpdp`");
+/*          
+          $lastPharmacy = sqlQuery("SELECT id FROM `pharmacies` ORDER BY id DESC LIMIT 1 ");
+           $sqlPharmacy = "INSERT INTO `pharmacies` (`id`, `name`, `transmit_method`, `email`, `ncpdp`, `npi`) VALUES
+                           (?, 'Test Direct Pharmacy', 1, NULL, 12345678, 321),
+                           (?, 'CCS Medical', 1, NULL, 1086203, 0)";
+            $addPharmacy = array($lastPharmacy+1, $lastPharmacy+2);
+           sqlStatement($sqlPharmacy, $addPharmacy);
+*/
```

Comment:
The thought that was being wrestled with is because this is an add on feature and may not be used. Why burden the system with having to carry this information in these scripts and database changes. I do understand keeping all this information in one central store. But the system allows for forms not to be stored in the central database.sql?
They can be moved. The other thought was so testing of the entries would be easier in my mind.